lib/cmd-start.js

```diff
@@ -16,7 +16,7 @@ function start (entry, opts) {
     })
   })
 
-  getPort(8080, 9000, function (err, port) {
+  getPort({port: 8080}).then(function (err, port) {
```

Comment:
just one last thing: these should be `.then(function (port) { }).catch(function (err) {})`

Reply:
I should not try to fix things while in holidays \o/